### PR TITLE
Updated weapons. Added weapon

### PR DESCRIPTION
**Pull request by wassupninja**
Added special concealable weapon. Hidden Blade

### CODE REVIEW

**Comment by WJohn** [comment]
1 force is 1 damage. This might as well not exist. Punching is way, way more effective.

**Comment by neersighted** [comment]
This is also not how you add a weapon. It will exist but not as you intend, it will not extend or retract or even update sprites.

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -214,6 +214,18 @@
 	shard.Consume()
 	T.ChangeTurf(T.baseturf)
 	T.CalculateAdjacentTurfs()
+	
+/obj/item/weapon/melee/knife/hidden_blade
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/melee/misc.dm]: As a child of knife. Its best feature is that you can butcher with it.

**Comment by PKPenguin321** [comment]
you've made some grievous errors and seem to be very new to this. i recommend messing around in a local server as an admin and doing some var-editing so you understand what stuff like "force" and "w_class" does.
if you want your thing to be retractable, make it a child of telebatons, not knives.

**Comment by Bawhoppen** [comment]
@PKPenguin321 >child of telebaton instead of switchblade

**Comment by PKPenguin321** [comment]
tru

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -214,6 +214,18 @@
...
 	T.ChangeTurf(T.baseturf)
 	T.CalculateAdjacentTurfs()
+	
+/obj/item/weapon/melee/knife/hidden_blade
+	name = "hidden blade"
+	desc = "A small yet sharp weapon, can be concealed when retracted."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "telebaton_0"
+	item_state = null
+	slot_flags = SLOT_GLOVES
```
Comment (duncathan) [inline code/game/objects/items/weapons/melee/misc.dm]: Alright this pr is bad but this line right here is alright

Comment (KazeEspada) [inline code/game/objects/items/weapons/melee/misc.dm]: You can attach it to your hands!

**Pull request closed without merging**